pom.xml

```diff
@@ -70,13 +70,14 @@
                 </exclusion>
             </exclusions>
         </dependency>
-
+        <!--
```

Comment:
If we don't need this section, why not just delete it. If we need it, we can look back in history to find it.

Reply:
Removed.